scope/src/doctor/commands/run.rs

```diff
@@ -173,6 +173,7 @@ fn transform_inputs(found_config: &FoundConfig, args: &DoctorRunArgs) -> RunTran
 
         let container = GroupActionContainer {
             group_name: group.metadata.name().to_string(),
+            group: group.clone(),
             actions: action_runs,
             additional_report_details: group.extra_report_args.clone(),
```

Comment:
We can propably use the group to access these things instead.
We're already cloning the group, there's no sense in also cloning other things we get out of the group.